feature-styles/HY_P_WaterbodiesPersistence.se

```diff
@@ -48,7 +48,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
     <se:Description>
-      <se:Title>standing waters persistence: perennial</se:Title>
+      <se:Title>Standing waters persistence: perennial</se:Title>
```

Comment:
For other cases with different geometries, the geometry was mentioned in the title - maybe align?